scilpy/tracking/propagator.py

```diff
@@ -387,15 +413,17 @@ def _get_sf(self, pos):
         Parameters
         ----------
         pos: ndarray (3,)
-            Position in voxmm in the trackable dataset.
+            Position in the trackable dataset, either in vox or voxmm space.
```

Comment:
The "either in vox or voxmm space" is a little bit misleading. It gives the impression we can supply it in any space. I'd just remove it, the position is in the space the propagator is, and the user should know that.

Reply:
Ok, thanks